internal/common/rules/ExampleMatrixRegistration.go

```diff
@@ -0,0 +1,30 @@
+package rules
```

Comment:
https://stackoverflow.com/questions/25161774/what-are-conventions-for-filenames-in-go/25162021#25162021

nit for file naming convention :)